contracts/Market.sol

```diff
@@ -224,7 +229,10 @@ contract Market is PausableAccessPreset {
     bytes32 s
   ) external whenNotPaused {
     uint256 certificateAmount = this.certificateAmountFromPurchaseTotal(amount);
-    _checkSupply({certificateAmount: certificateAmount});
+    _validatePrioritySupply({
```

Comment:
@amiecorso not sure why this function wasn't previously validating priority supply, but wanted to run it by you that this is still doing the correct thing. I merged _checkSupply with this function since they did nearly the same thing. AFAIK the priority threshold should still be validated against regular checkouts.